bundle-generator/deployment/src/main/java/io/quarkiverse/operatorsdk/bundle/deployment/BundleProcessor.java

```diff
@@ -127,6 +139,37 @@ CSVMetadataBuildItem gatherCSVMetadata(KubernetesConfig kubernetesConfig,
         return new CSVMetadataBuildItem(csvGroups);
     }
 
+    private static String getDefaultProviderURLFromSCMInfo(ApplicationInfoBuildItem appConfiguration,
+            JarBuildItem jarBuildItem) {
+        final var maybeProject = KubernetesCommonHelper.createProject(appConfiguration, Optional.empty(),
+                jarBuildItem.getPath());
+        return maybeProject.map(project -> {
+            final var scmInfo = project.getScmInfo();
+            if (scmInfo != null) {
+                var origin = scmInfo.getRemote().get("origin");
```

Comment:
Again.. same as with that CVE. We should make sure that this can be explicitly overridable -- there are cases when origin might point actually be a fork which one doesn't want to publicly advertise  (e.g. when builds are for whatever reason using a mirror URL or when you simply have manual steps in your workflow)

Reply:
I absolutely must echo @jcechace concern here.  In almost all use cases, `origin` for me is my fork, and `upstream` refers to the public repository.

Reply:
Yes, it's very tricky so maybe I'll just remove this and let people deal with it manually instead…

Reply:
> Again.. same as with that CVE. We should make sure that this can be explicitly overridable -- there are cases when origin might point actually be a fork which one doesn't want to publicly advertise (e.g. when builds are for whatever reason using a mirror URL or when you simply have manual steps in your workflow)

Note that this is for the default value and is overridable using `@CSVMetadata` and its `provider` field. That said, this computed value might be faulty in many situations so it's best to remove it.